### DISCUSSION TITLE
Daemonization in Docker

### DISCUSSION THREAD

**Discussion by langBeralarm** [celery/celery#8638] [Q&A]
Is it recommended to run Celery as a Daemon in Docker if it's the only process running in this Docker Container?
As far as I can gather it doesn't have an impact on performance but since the recommended way to run Celery in production is as a Daemon I just want to make sure I'm not jumping to conclusions.

**Comment by auvipy** [comment]
as far know, docker do have daemon as well https://docs.docker.com/config/daemon/

**Comment by Kotsakis** [comment]
Any opinions on that?
